tests/cli/test_cli.py

```diff
@@ -188,6 +188,20 @@ def test_cli_rbac_webserver_debug(self):
         p.terminate()
         p.wait()
 
+    def test_cli_webserver_flask_debug(self):
+        env = os.environ.copy()
+        env['AIRFLOW__WEBSERVER__RBAC'] = 'True'
+        env['AIRFLOW__FLASK_WEB__PREFERRED_URL_SCHEME'] = 'https'
+        p = psutil.Popen(["airflow", "webserver", "-d"], env=env)
```

Comment:
This test doesn't really assert that we've configured the webserver right. Better would be to use mocking